docs/networking.md

```diff
@@ -18,6 +18,12 @@ If you don't install CoreDNS, you will need to install a cluster DNS provider yo
 
 CoreDNS is deployed with the [autoscaler](https://github.com/kubernetes-incubator/cluster-proportional-autoscaler) by default. To disable it or change its config, use the [HelmChartConfig](https://docs.rke2.io/helm/#customizing-packaged-components-with-helmchartconfig) resource.
 
+### NodeLocal DNSCache
+
+[NodeLocal DNSCache](https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/) improves the performance by running a dns caching agent on each node. It is possible to activate this feature by using HelmChartConfig and updating the value `.Values.nodelocal.enabled` as can be seen in its [Chart](https://github.com/rancher/rke2-charts/blob/main/charts/rke2-coredns/rke2-coredns/1.16.201-build2021072302/values.yaml#L305-L311). The helm controller will redeploy coredns with the new config. Please be aware that nodelocal modifies the iptables of the node to intercept DNS traffic. Therefore, activating and then deactivating this feature without redeploying, will cause the DNS service to stop working.
```

Comment:
Rather than linking to the values file, can you provide an example HelmChartConfig manifest to do this, similar to the one at https://github.com/rancher/rke2/blob/master/docs/helm.md#customizing-packaged-components-with-helmchartconfig

Reply:
Yes, makes more sense. They user will appreciate that more